copy.go

```diff
@@ -249,12 +249,13 @@ func copyGraph(ctx context.Context, src content.ReadOnlyStorage, dst content.Sto
 			for _, node := range successors {
 				done, committed := tracker.TryCommit(node)
 				if committed {
-					return fmt.Errorf("%s: %s: successor not committed", desc.Digest, node.Digest)
+					err := fmt.Errorf("%s: %s: successor not committed", desc.Digest, node.Digest)
+					return newCopyError("copyGraph", CopyErrorOriginInternal, err)
 				}
 				select {
 				case <-done:
 				case <-ctx.Done():
-					return ctx.Err()
+					return newCopyError("copyGraph", CopyErrorOriginInternal, ctx.Err())
```

Comment:
"copyGraph" is internal. It is not that meaningful to end users.

Reply:
refactored